### PR TITLE
implementatie contact-api

### PR DESCRIPTION
**Pull request by MarcelWildenburg**
School/GastDocent kunnen vanuit Zoek scherm contact met elkaar opnemen.

### CODE REVIEW

File: src/app/components/contact-modal/contact-modal.component.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+      return firstValueFrom(sendEmailCall)
+        .then(() => {
+          this.hideForm = true;
+          this.hideConfirmation = false;
+          return true;
+        })
+        .catch((error: any) => {
+          console.log(error);
+          console.error('send email error');
```
Comment (maritaria) [inline src/app/components/contact-modal/contact-modal.component.ts:86]: Shouldn't this go through angular's error pipeline?

File: src/app/components/contact-modal/contact-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          return false;
+        });
+      });
+  }
+
+  public closeModal() {
+    this.close();
+  }
```
Comment (maritaria) [inline src/app/components/contact-modal/contact-modal.component.ts:94]: This doesn't seem to do anything, wouldn't it just be `close()` everywhere it now says `closeModal()`?

File: src/app/services/littil-contact/littil-contact.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  beforeEach(() => {
+    spectator = createHttp();
+  });
+
+  it('should be created', () => {
+    expect(spectator.service).toBeTruthy();
+  });
+
+  describe('sendEmail', () => {
+    it('should send email', () => {
```
[the remark this answers is not in the document]
Comment (MarcelWildenburg) [reply src/app/services/littil-contact/littil-contact.service.spec.ts:24]: Change accoording to tests in other services

File: src/app/components/contact-modal/contact-modal.component.ts (hunk 1 above)
Comment (MarcelWildenburg) [reply src/app/components/contact-modal/contact-modal.component.ts]: changed text that is logged in the console.
Logging to console is similar with other solutions in the frontend.

File: src/app/components/contact-modal/contact-modal.component.ts (hunk 2 above)
Comment (MarcelWildenburg) [reply src/app/components/contact-modal/contact-modal.component.ts]: closeModal() replaced with close()

File: src/app/components/contact-modal/contact-modal.component.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
+<div [@hideShow]="loading || hideForm ? 'hidden' : 'visible'" class="prose">
```
[the remark this answers is not in the document]
Comment (MarcelWildenburg) [reply src/app/components/contact-modal/contact-modal.component.html:1]: How would you change this ?

File: src/app/services/littil-contact/littil-contact.service.spec.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    imports: [HttpClientTestingModule],
+    providers: [ContactService],
+  });
+
+  beforeEach(() => {
+    spectator = createHttp();
+  });
+
+  it('should be created', () => {
+    expect(spectator.service).toBeTruthy();
```
[the remark this answers is not in the document]
Comment (MarcelWildenburg) [reply src/app/services/littil-contact/littil-contact.service.spec.ts:20]: correct. This is similar setup as used in other tests for service.

File: src/app/components/contact-modal/contact-modal.component.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (LDMGN) [reply src/app/components/contact-modal/contact-modal.component.html:1]: Create https://devoxx4kids.atlassian.net/browse/LITTIL-263

**Review by EvelienArts — APPROVED** [review]

**Pull request merged**